ci/test_wheel_ucxx.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+PROJECT_NAME="ucxx"
+
+source "$(dirname "$0")/test_utils.sh"
+source "$(dirname "$0")/test_common.sh"
+
+mkdir -p ./dist
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="${PROJECT_NAME}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+
+# echo to expand wildcard before adding `[extra]` requires for pip
+python -m pip install $(echo ./dist/${PROJECT_NAME}*.whl)[test]
+
+# Run smoke tests for aarch64 pull requests
```

Comment:
Same question about smoke tests. Implement them, or remove the commented section.

Reply:
Also done in https://github.com/rapidsai/ucxx/pull/167/commits/3ef814b2ca7340d9fa4b6fda3f4624ed2d7cd323 .